asf_heat_pump_suitability/config/base.yaml

```diff
@@ -11,6 +11,7 @@ data_source:
   GB_ons_garden_space_access: "s3://asf-heat-pump-suitability/source_data/ONS_Apr2020_access_to_garden_space.xlsx"
   GB_osopen_uprn_latlon: "s3://asf-heat-pump-suitability/source_data/osopenuprn_202405_csv.zip"
   EW_census_accommodation_type: "s3://asf-heat-pump-suitability/source_data/2021census_Mar2023_accommodation_type_E_W.csv"
+  UK_spa_offgasgrid: "s3://asf-heat-pump-suitability/source_data/2024_vMar2024_SPA_offgaspostcode_UK.xlsx"
```

Comment:
Can you add info about this data to the features table in the config/README.md file? I didn't think about this in the property density PR - but you will need to add info about this data too (can just add it all in this PR though)

Reply:
I've done this for the number of households, land area, off gas grid and also listed buildings.